csharp/ql/src/semmle/code/csharp/ir/implementation/raw/internal/TranslatedDeclarationEntry.qll

```diff
@@ -60,7 +60,7 @@ class TranslatedNonVariableDeclarationEntry extends TranslatedDeclarationEntry {
       Type resultType, boolean isLValue) {
     opcode instanceof Opcode::NoOp and
     tag = OnlyInstructionTag() and
-    resultType instanceof VoidType and
+    resultType instanceof Language::UnknownType and
```

Comment:
This one should be `VoidType`, because the instruction does not return a result at all. `Language::UnknownType` is for results that exist, but whose type is not known.